src/app/shared/components/configuration/configuration-confirmation-dialog/configuration-confirmation-dialog.component.ts

```
@@ -22,17 +22,28 @@
+
+  @Input() redirectLink: string;
+
   @Output() warningAccepted = new EventEmitter<ConfigurationWarningOut>();
 
   readonly brandingConfig = brandingConfig;
 
+  AppName = AppName;
+
+  brandIcon: string;
+
   constructor() { }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)